package.json

```diff
@@ -3,44 +3,34 @@
   "version": "1.0.0",
   "description": "",
   "dependencies": {
+    "@uswds/compile": "^1.0.0-beta.3",
+    "@uswds/uswds": "3.0.2",
     "accessible-autocomplete": "^1.6.2",
-    "autoprefixer": "^10.4.0",
     "browserify": "^16.3.0",
     "concurrently": "^6.4.0",
     "css-mqpacker": "^7.0.0",
     "cssnano": "^5.0.12",
     "fuse.js": "^3.4.5",
     "glossary-panel": "^1.0.0",
     "gulp": "^4.0.2",
-    "gulp-dart-sass": "^1.0.2",
-    "gulp-notify": "^4.0.0",
-    "gulp-postcss": "^9.0.1",
-    "gulp-rename": "^2.0.0",
-    "gulp-replace": "^1.0.0",
-    "gulp-sourcemaps": "^3.0.0",
-    "gulplog": "^1.0.0",
     "js-yaml": "^3.13.1",
     "lodash.assign": "^4.2.0",
     "pa11y-ci": "^2.3.0",
     "path": "^0.12.7",
     "postcss": "^8.4.4",
-    "sass": "^1.44.0",
+    "sass": "^1.54.0",
     "smooth-scroll": "^16.1.0",
     "underscore": "^1.13.1",
-    "uswds": "^2.0.3",
-    "uswds-gulp": "github:uswds/uswds-gulp",
     "vinyl-buffer": "^1.0.1",
     "vinyl-source-stream": "^2.0.0",
     "watchify": "^4.0.0"
   },
   "scripts": {
-    "build": "gulp init",
+    "build": "npx gulp updateUswds",
```

Comment:
I know this copies assets and compiles, but it might be more straightforward to separate these out via copyAssets and then compile. 

Even adding a comment to the gulpfile (above line 23) would help explain what's happening.

Reply:
Fair point, I've added some notes and additional build step for `copyAssets`.